scripts/kvm-test.py

```diff
@@ -236,16 +219,17 @@ def mounter(src, dest):
         run(f'sudo umount {dest}')
 
 
+def livefs_edit(ctx, *args):
+    run(f'sudo PYTHONPATH=$LIVEFS_EDITOR python3 -m livefs_edit \
+          {ctx.baseiso} {ctx.iso} {*args}')
```

Comment:
I was curious about the how the `*args` would be joined together in a f-string.
But I'm not able to run this (at least on Python 3.9):
```python
    (*args)
     ^
SyntaxError: f-string: can't use starred expression here
```
Is this something new?

Reply:
ah, that fails for me also, clearly that path wasn't tested